apps/emqx_cluster_link/src/emqx_cluster_link_schema.erl

```diff
@@ -46,13 +49,10 @@ fields("link") ->
             default => #{<<"enable">> => false},
             desc => ?DESC("ssl")
         }},
-        %% TODO: validate topics:
-        %% - basic topic validation
-        %% - non-overlapping (not intersecting) filters?
-        %%  (this may be not required, depends on config update implementation)
         {topics,
             ?HOCON(?ARRAY(binary()), #{required => true, validator => fun topics_validator/1})},
-        {pool_size, ?HOCON(pos_integer(), #{default => emqx_vm:schedulers() * 2})}
+        {pool_size, ?HOCON(pos_integer(), #{default => emqx_vm:schedulers() * 2})},
+        {ps_actor_incarnation, ?HOCON(integer(), #{default => 0, importance => ?IMPORTANCE_HIDDEN})}
```

Comment:
Nit. I think a short comment describing what it is for would be really helpful.

Reply:
Added.